compiler/pipes/collect-main-edges.cpp

```diff
@@ -18,7 +18,45 @@
 #include "compiler/inferring/restriction-non-void.h"
 #include "compiler/inferring/type-node.h"
 #include "compiler/type-hint.h"
+#include "common/php-functions.h"
 
+namespace {
+
+SwitchKind get_switch_kind(VertexAdaptor<op_switch> s) {
+  bool all_cases_are_ints = true;
```

Comment:
maybe, rewrite to `num_const_int_cases` to be similar to others?

Reply:
Done.